functions.php

```diff
@@ -47,6 +47,11 @@
 add_image_size( 'landscape-medium', 740, 416, true );
 add_image_size( 'landscape-featured', 1000, 500, true );
 
+//Media Upload size
+@ini_set( 'upload_max_size' , '64M' );
+@ini_set( 'post_max_size', '64M');
+@ini_set( 'max_execution_time', '300' );
+
```

Comment:
These configuration variables should typically be set at the project level rather than in a theme or plugin.

```suggestion
```